src/lib/commercial-features.spec.ts

```diff
@@ -105,36 +106,36 @@ describe('Commercial features', () => {
 	describe('DFP advertising', () => {
 		it('Runs by default', () => {
 			const features = new CommercialFeatures();
-			expect(features.dfpAdvertising).toBe(true);
+			expect(features.shouldLoadGoogletag).toBe(true);
```

Comment:
I don't mind if you want to stick with 'should' but this is a bit shorter?

```suggestion
			expect(features.loadGoogletag).toBe(true);
```

Reply:
I think I'll leave it as is for now, just to keep it aligned with the switch. I do appreciate it's a little long though, so maybe we can rename it across Frontend/Commercial once this has settled?